src/WebJobs.Script.WebHost/Handlers/WebScriptHostHandler.cs

```diff
@@ -10,19 +11,20 @@
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Handlers
 {
-    public class EnsureHostRunningHandler : DelegatingHandler
+    public class WebScriptHostHandler : DelegatingHandler
```

Comment:
Initially I renamed this because I was adding additional logic to it, but I reverted all that. However, I like the new name better, so I kept it.